tests/testthat/test-auto-report-functions.R

```diff
@@ -90,6 +90,10 @@ test_that("A per-user subscription table (for shiny) can be made", {
   expect_true(is.list(makeUserSubscriptionTab(session = shinySession)))
 })
 
+test_that("A per-user subscription table (for shiny) can be made", {
```

Comment:
Egen test og burde ha eget navn, eksempelvis "A per-user subscription table __V2__ (for shiny) can be made